src/Core/Manifest/ClassLoader.php

```diff
@@ -120,13 +120,14 @@ public function getItemPath($class)
      *
      * @param bool $includeTests
      * @param bool $forceRegen
+     * @param string[] $ignoredCILibs
      */
-    public function init($includeTests = false, $forceRegen = false)
+    public function init($includeTests = false, $forceRegen = false, array $ignoredCILibs = [])
```

Comment:
I had to change a bunch of `init` and `regenerate` methods to add a `$ignoredCILibs` parameters.

In the past, we didn't consider this a breaking API change.